pylzy/lzy/api/v1/__init__.py

```diff
@@ -41,22 +41,23 @@
 # pylint: disable=[invalid-name]
 # noinspection PyShadowingNames
 def op(
-    func: Optional[FuncT] = None,
```

Comment:
new codestyle, use 4 spaces, plz

Reply:
why are all codestyle checks are green in that case?

Reply:
changed local codestyle settings